src/main/java/mate/academy/spring/service/mapper/UserMapper.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy.spring.service.mapper;
 
+import mate.academy.spring.dto.request.UserRequestDto;
 import mate.academy.spring.dto.response.UserResponseDto;
 import mate.academy.spring.model.User;
 import org.springframework.stereotype.Component;
 
 @Component
-public class UserMapper implements ResponseDtoMapper<UserResponseDto, User> {
+public class UserMapper implements RequestDtoMapper<UserRequestDto, User>,
+        ResponseDtoMapper<UserResponseDto, User> {
+
     @Override
     public UserResponseDto mapToDto(User user) {
         UserResponseDto responseDto = new UserResponseDto();
         responseDto.setId(user.getId());
         responseDto.setEmail(user.getEmail());
         return responseDto;
     }
+
+    @Override
+    public User mapToModel(UserRequestDto dto) {
```

Comment:
where do you use this method?

Reply:
Yes, now nowhere)) (it was in processing "my modified" of UserDao... but my additional methods were in AbstructDao))